os/net/netmgr/netdev_lwip.c

```diff
@@ -511,12 +511,27 @@ static int lwip_set_ip6addr(struct netdev *dev, struct sockaddr_storage *addr, i
 
 	return 0;
 }
+
+static int lwip_set_ip6addr_type(struct netdev *dev, uint8_t type)
+{
+	struct netif *ni = GET_NETIF_FROM_NETDEV(dev);
+	ni->ip6_addr_type = type;
```

Comment:
How about checking `ni` is NULL or not?

Reply:
every netdev contain netif. it's guaranteed.